src/shared-with-pluot-core/Util.js

```diff
@@ -0,0 +1,27 @@
+export const getParticipantIsSubscribedToTrack = (state, id, mediaTag) => {
+  return _getIsSubscribedToTrack(state,
+                                 state.participants[id],
+                                 state.local.public.id,
+                                 mediaTag);
+}
+
+export const getLocalIsSubscribedToTrack = (state, id, mediaTag) => {
+  return _getIsSubscribedToTrack(state,
+                                 state.local,
+                                 id,
+                                 mediaTag);
+}
+
+export const _getIsSubscribedToTrack = (state, p, p2id, mediaTag) => {
+  // if we don't have a participant record at all, assume that 
+  // true is the safest thing to return, here
```

Comment:
hm, code seems to disagree with comment

Reply:
agreed. also, code seems more correct to me?